serving/docs/lmi/deployment_guide/configurations.md

```diff
@@ -125,17 +125,20 @@ You can find these configurations in the respective [user guides](../user_guides
 
 All LMI Configuration keys available in the `serving.properties` format can be specified as environment variables.
 
-The translation for `engine` is unique. The configuration `engine=<engine>` is translated to `SERVING_LOAD_MODELS=test::<engine>=/opt/ml/model`.
-For example:
+The property `option.model_id` is unique. It is translated to `HF_MODEL_ID`.
```

Comment:
option_model_id and HF_MODEL_ID  are different thing, we might want to document HF_MODEL_ID separately